packages/lib/src/index.ts

```diff
@@ -3,8 +3,9 @@ import * as typeIs from './typeIs';
 import createUniqueClassName from './createUniqueClassName';
 import getNodeTextContent from './getNodeTextContent';
 import DarkModeProps, { Theme, ThemeType } from './DarkModeProps';
+import getTheme from './getTheme';
```

Comment:
Should've caught this on the pervious PR. Why do we have both 
`Thème` and `ThemeType`. We should be able to use just one name `Theme` for both the object and type